Demo/Demo/Features/FeatureBaseViewController.swift

```diff
@@ -0,0 +1,100 @@
+import UIKit
+
+class FeatureBaseViewController: UIViewController {
+
+    private enum Constants {
+        static let stackViewSpacing: CGFloat = 16
+        static let layoutSpacing: CGFloat = 16
+    }
+
+    lazy var stackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .horizontal
+        view.alignment = .center
+        view.distribution = .fillEqually
+        view.spacing = Constants.stackViewSpacing
+        return view
+    }()
+
+    lazy var amountTextField: CustomTextField = {
+        let textField = CustomTextField(placeholderText: "Amount", defaultValue: "10.00")
+        textField.keyboardType = .decimalPad
+        return textField
+    }()
+
+    lazy var createOrderButton: CustomButton = {
+        let createOrderButton = CustomButton(title: "Create Order")
+        createOrderButton.addTarget(self, action: #selector(createOrderTapped), for: .touchUpInside)
+        createOrderButton.tintColor = .systemPink
+        return createOrderButton
+    }()
+
+    lazy var titleLabel: UILabel = {
```

Comment:
Nitpick - naming. Title sounds like it's the top of the screen. Maybe `bottomStatusLabel` or something